### PR TITLE
Remove old design proposals which have been archived

### PR DESCRIPTION
**Pull request by rytswd**
As mentioned in each file, remove old design proposals as it has passed 2022-04-01.

This is the follow-up of #6258.

### CODE REVIEW

File: contributors/design-proposals/README.md [hunk 1]
```diff
@@ -1,9 +0,0 @@
-Design proposals have been archived.
-
-To view the last version of this document, see the [Design Proposals Archive Repo](https://github.com/kubernetes/design-proposals-archive/).
-
-For information on the replacement design proposal process, see the [Kubernetes Enhancement Proposals (KEP)](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/0000-kep-process) process.
-
-
-Please remove after 2022-04-01 or the release of Kubernetes 1.24, whichever comes first.
```
[the remark this answers is not in the document]
Comment (rytswd) [reply contributors/design-proposals/README.md:8]: Most certainly, I agree that would be more friendly for people finding the old doc like I did. I will update the PR by the EOD today 👍

[the remark this answers is not in the document]
Comment (rytswd) [reply contributors/design-proposals/README.md:8]: @mrbobbytables Sorry for the delay, I have updated the PR with the following:

- b141b17 - Remove old files
- f301f3a - Adjust the top level README content

**Review by mrbobbytables — COMMENTED** [review]
No need to be sorry :) Things normally take days here :P 
Thanks for cleaning this up!

/approve
/hold
just for a 2nd set of eyes 👍

**Pull request merged**